python/graphstorm/gconstruct/transform.py

```diff
@@ -507,7 +604,17 @@ def parse_label_ops(confs, is_node):
     label_conf = confs[0]
     assert 'task_type' in label_conf, "'task_type' must be defined in the label field."
     task_type = label_conf['task_type']
-    # By default, we use all labels for training.
+    if 'custom_split' in label_conf:
```

Comment:
Can you add a docstring or modify the dataloading rst file to explain the format of custom_split?

Reply:
which rst file are you talking about?

Reply:
https://github.com/awslabs/graphstorm/blob/main/docs/source/tutorials/own-data.rst